dev/com.ibm.ws.ejbcontainer.core/src/com/ibm/ejs/container/BeanMetaData.java

```
@@ -1129,6 +1129,12 @@ public int getActivationPolicy() {
      */
     public boolean ivHasAsynchMethod = false;
 
+    /**
```

While this works, BeanMetaData is not really the proper place to keep track of error only data.  We work to not bloat BeanMetaData since it remains in memory for the life of the application. Since this is an error only scenario, we should just throw the error when it is found.

I see that this follows a similar pattern as javax.ejb.Asynchronous, but the field is used for working configurations, not just for detecting errors.

Please just remove all changes from BeanMetaData, and instead throw the error when the annotation is identified.